<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcRenderDialog</class>
 <widget class="QDialog" name="lcRenderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="MainLayout">
     <item>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Output:</string>
         </property>
         <property name="buddy">
          <cstring>OutputEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="OutputEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="OutputBrowseButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="lcRenderPreviewWidget" name="preview" native="true"/>
     </item>
     <item>
      <widget class="QProgressBar" name="RenderProgress">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lcRenderPreviewWidget</class>
   <extends>QWidget</extends>
   <header>lc_renderdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>OutputEdit</tabstop>
  <tabstop>OutputBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
